### PR TITLE
Switch to rust-vmm-ci

### LINKED ISSUE #4: Switch to rust-vmm-ci
Use rust-vmm-ci as a gitsubmodule to test the code. Since linux-loader has non-default features, a custom pipeline is also required.

For more details about how to make the switch, please check the [rust-vmm-ci readme](https://github.com/rust-vmm/rust-vmm-ci)

### PR DESCRIPTION
**Pull request by aghecenco**
Fixes #4 
* removed `rust-vmm-ci` pipeline, using the one from the submodule instead
* updated the container to v5 on the internal custom pipeline
* fixed the remaining failed tests

(Still not very sure of what I'm doing, please call out any stupid stuff)

### CODE REVIEW

**Comment by aghecenco** [comment]
@andreeaflorescu the build is successful, but the webhook didn't get the message, help? :smiley: 
https://buildkite.com/rust-vmm/linux-loader-ci/builds/70

**Comment by aghecenco** [comment]
> @andreeaflorescu the build is successful, but the webhook didn't get the message, help? smiley
> https://buildkite.com/rust-vmm/linux-loader-ci/builds/70

This was some draft PR black magic. Once updated to ready-for-review PR, the correct status is reported.

File: README.md [hunk 1]
```diff
@@ -18,16 +18,17 @@ cargo build
 Our Continuous Integration (CI) pipeline is implemented on top of
```
Comment (andreeaflorescu) [inline README.md]: Hmm, the readme should be updated as well now that we use the rust-vmm-ci directly. In other repositories we just linked the rust-vmm-ci and got rid of all the other details.

Comment (aghecenco) [reply README.md]: Done :heavy_check_mark:

**Comment by aghecenco** [comment]
> The tests directory needs to be removed as well as we are using the coverage test from rust-vmm-ci.

Done

**Pull request merged**